app/uk/gov/hmrc/testuser/models/TestUser.scala

```diff
@@ -59,6 +60,7 @@ object Services extends Seq[Service] {
     Service(ServiceKeys.RELIEF_AT_SOURCE, "Relief at Source", Seq(ORGANISATION)),
     Service(ServiceKeys.CUSTOMS_SERVICES, "Customs Services", Seq(INDIVIDUAL, ORGANISATION)),
     Service(ServiceKeys.GOODS_VEHICLE_MOVEMENTS, "Goods Vehicle Services", Seq(INDIVIDUAL, ORGANISATION)),
+    Service(ServiceKeys.CTC_LEGACY, "Common Transit Convention Traders Legacy", Seq(INDIVIDUAL, ORGANISATION)),
     Service(ServiceKeys.CTC, "Common Transit Convention Traders", Seq(INDIVIDUAL, ORGANISATION)),
```

Comment:
I think the `allowedUserTypes` should probably only be `ORGANISATION` to match the enrolment config.

Reply:
FWIW the same is true for the legacy enrolment too https://github.com/hmrc/service-enrolment-config/blob/365f7bbc359351470b9714330275fb51fcf72b98/conf/SEC1_with_enrolment_rules_json/prod/HMCE-NCTS-ORG.json

Reply:
Obviously this is superseded now by our latest requirements 😆